tools/cpp/windows_cc_configure.bzl

```diff
@@ -69,8 +113,14 @@ def _get_escaped_windows_msys_starlark_content(repository_ctx, use_mingw = False
 
 def _get_system_root(repository_ctx):
     """Get System root path on Windows, default is C:\\\Windows. Doesn't %-escape the result."""
-    if "SYSTEMROOT" in repository_ctx.os.environ:
-        return escape_string(repository_ctx.os.environ["SYSTEMROOT"])
+    systemroot = _get_path_env_var(
+        repository_ctx,
+        "SYSTEMROOT",
+        default = "__not_found__",
+        enable_warning = False,
```

Comment:
Can we set default to None here? If so, we don't need enable_warning = False, either.

Reply:
We can't, because then `_get_path_env_var` fail when SYSTEMROOT is undefined.

Reply:
But we could just set default to `c:\windows`, let me do that.

Reply:
Sounds good, but for "BAZEL_VC" and "BAZEL_VS", I guess we can set the default to None? It will make the code a bit cleaner.

Reply:
Done.

Reply:
You're right! Done. Much nicer indeed!